client/components/jetpack/threat-dialog-new/index.tsx

```diff
@@ -74,15 +62,12 @@ const ThreatDialog: React.FC< Props > = ( {
 		>
 			<>
 				<p>
-					{ action === 'fix' &&
-						translate( 'Jetpack will fix the %(severity)s threat:', {
-							args: { severity: getThreatSeverityText( threat ) },
-						} ) }
+					{ action === 'fix' && translate( 'Jetpack will fix the threat:' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'This is the threat Jetpack will fix:', 'These are the threats Jetpack will fix:' )` *ES Score: 7*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/more-informative-scan/fix-header-overflow)